doc/cygnus-ngsi-ld/flume_extensions_catalogue/ngsi_ckan_sink.md

```diff
@@ -0,0 +1,421 @@
+# <a name="top"></a>NGSICKANSink
```

Comment:
Is there a master index of sinks for cynus-ngsi-ld in which the new sink for CKAN should be mentioned?

I don't remember very well, but I think we have something like that (maybe in some README.md file?)

Reply:
The master index of the sinks is inside of the flume_extensions_catalogue directory. I have updated this readme file for including the new sink.